marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping-functions/getMappingFunctions.sjs

```diff
@@ -0,0 +1,11 @@
+const mappingFunctions = require('/data-hub/5/extensions/mappingFunctions.sjs');
+const test = require("/test/test-helper.xqy");
+
+function testMappingFunctions() {
+return [
+    test.assertEqual(129, Object.keys(mappingFunctions.getXpathFunctions()).length),
+    test.assertEqual(4, Object.keys(mappingFunctions.getMarkLogicFunctions()).length)
```

Comment:
I think an assertion message is neeed here too to explain why 4 functions are expected - i.e. state that we expect there to be 4 functions in the "DHF core" namespace.